addons/im_livechat/static/tests/tours/im_livechat_pivot_redirect_tour.js

```diff
@@ -0,0 +1,37 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("im_livechat_pivot_redirect_tour", {
+    steps: () => [
+        {
+            isActive: ["enterprise"],
+            content: "open command palette",
+            trigger: ".o_home_menu",
+            run: "click && press ctrl+k",
+        },
```

Comment:
What's the need for this step? IMO the next step will work fine for both community and enterprise.